components/layouts/featured/featured.hbs

```diff
@@ -37,7 +37,7 @@
       </div><!-- .content-block__upper -->
 
       <div class="content-block__lower">
-        <p class="content-block__subtitle">2021 Salary Guide</p>
+        <div class="content-block__subtitle">2021 Salary Guide</div>
```

Comment:
I'm not quite sure that markup would be screen reader friendly.
What is the rationale for removing `<p>` tag?

Reply:
Strictly speaking, I don't think it changes the outcomes here aside from semantics. It technically isn't paragraph type content. A div or span should be fine in the long run.

Reply:
I do notice that passing dotCMS variable fields can sometime means more than just a string. Could be another `p tag` which mess up the elements a bit. also I think there was some default margin on `p tag` that I didn't want to mess with at the time. removed that margin rule though.